src/Options.php

```diff
@@ -815,7 +815,7 @@ private function configureOptions(OptionsResolver $resolver): void
                 return $event;
             },
             'tags' => [],
-            'error_types' => E_ALL,
+            'error_types' => error_reporting(),
```

Comment:
We should also document this change in the upgrade docs since this might be unexpected.

Reply:
Nice catch. I would like to open a discussion about retaining this option, but for the moment I agree that this change should be documented